### DISCUSSION TITLE
[BUG] Error when trying to install.

### DISCUSSION THREAD

**Discussion by ghost** [Serene-Arc/bulk-downloader-for-reddit#530] [General]
- [ ] I am reporting a bug.
- [x] I am running the latest version of BDfR
- [x] I have read the [Opening an issue](https://github.com/aliparlakci/bulk-downloader-for-reddit/blob/master/docs/CONTRIBUTING.md#opening-an-issue)
  
## Description

When trying to install, it gives the following error:
ERROR: Cannot install bdfr==2.0.0, bdfr==2.1.0, bdfr==2.1.1, bdfr==2.2.0, bdfr==2.3.0, bdfr==2.4.0 and bdfr==2.4.1 because these package versions have conflicting dependencies.
How do I specify which version I want to upgrade to? (If that even is a possibility, I'm a complete idiot and know next to nothing of coding)

## Command
```
Paste here the command(s) that causes the bug
py -m pip install bdfr --upgrade

```

## Environment (please complete the following information):
 - OS: [e.g. Windows 10]
 - Python version: [e.g. 3.9.4]

## Logs
```
Paste the log output here.
```

**Comment by ghost** [comment]
I've tried the same command but with bdfr-2.4.1, bdfr2.4.1 and bdfr 2.4.1 and none of them worked. Again, I want to put emphasis on the fact that I'm an idiot and know nothing of coding.

**Comment by Serene-Arc** [comment]
Please post the actual command and the actual output from the command when it errors out.
